proto/minder/v1/minder.proto

```diff
@@ -1023,6 +1023,8 @@ message Project {
     // display_names are short *non-unique* strings to provide
     // a user-friendly name for presentation in lists, etc.
     string display_name = 5;
+    bool is_organization = 8;
```

Comment:
Discussed offline, but I don't think we want to change our public API in this PR.